cmd/xsDeploy.go

```diff
@@ -110,7 +110,8 @@ xs {{.Mode.GetDeployCommand}} -i {{.OperationID}} -a {{.Action.GetAction}}
 
```

Comment:
The changes here are caused by updating the fileUtils. I can refactor this out into another PR in case that makes the review too hard.